pkg/controller/manager/clusternetwork/controller.go

```diff
@@ -137,6 +155,26 @@ func (h Handler) initializeClusterNetwork() error {
 	return nil
 }
 
+func (h Handler) initializeUntaggedNetwork() error {
+	bridgeName := iface.GenerateName(utils.ManagementClusterNetworkName, iface.BridgeSuffix)
+	untaggedNetworkConfig := fmt.Sprintf(networkConfigTemplate, untaggedNetworkName, bridgeName)
+	untaggedNetwork := &cniv1.NetworkAttachmentDefinition{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: publicNamespace,
```

Comment:
any reasons why it should be limited to `harvester-public` ns only and need a default config?

Reply:
Because all the users can access the `harvester-public` ns and the VMs across different namespaces can use the NAD in the `harvester-public` ns.

Reply:
I would prefer to leave the users/admins to create on-demand, exposing untagged-network on the public ns is not a good practice for security and RBAC concerns.

Reply:
Agree, let me delete the untagged network initialization and communicate with UI team.